docs/src/main/sphinx/connector/mariadb.md

```diff
@@ -261,6 +261,9 @@ to the following table:
       `explicit_defaults_for_timestamp
       <https://mariadb.com/docs/reference/mdb/system-variables/explicit_defaults_for_timestamp/>`_
       to avoid implicit default values and use `NULL` as the default value.
+* - `DATETIME(n)`
+  - `TIMESTAMP(n)`
+  -
```

Comment:
This section is Trino → MariaDB type mapping. Please move to "MariaDB type to Trino type mapping" section.

Reply:
ah yea. Fixed.